data-prepper-logstash-configuration/src/test/resources/org/opensearch/dataprepper/logstash/log-ingest-to-logstash-opensearch.expected.yaml

```diff
@@ -0,0 +1,18 @@
+logstash-converted-pipeline:
+  source:
+    http:
+      max_connection_count: 500
+      request_timeout: 10000
+  processor:
+    - grok:
+        match:
+          log:
+            - "%{COMBINEDAPACHELOG}"
+  sink:
+    - opensearch:
+        hosts:
+          - "fakedomain.us-east-1.es.amazonaws.com"
+        username: "myuser"
+        password: "mypassword"
+        index: "my-index"
+        insecure: false
```

Comment:
Nit: I would rather see an integration test with a non-default value for this, as it feels like there is some chance that it isn't actually taking into consideration and negating the value from `ssl_certificate_verification`

Reply:
Having two test cases here makes sense.

By the way, default values shouldn't end up in the generated YAML. So there is a good indication that this is working. But, I'm still for adding another input file.

Reply:
Thanks for the feedback. Yeah, as David mentioned default values won't be there in YAML files.
I'll add one more integration test.